source/installation/index.html

```diff
@@ -6,6 +6,25 @@
 toc: true
 ---
 
+<p>
+Home Assistant can be installed in 4 flavors:
+</p>
+<ul>
+    <li><em>Home Assistant Operating System</em>: Minimal Operating System optimized to power Home Assistant. Comes with Supervisor pre-installed. Recommended for new users.</li>
+    <li><em>Home Assistant Supervised</em>: Container based installation with our Supervisor to manage Home Assistant Core and Add-Ons.</li>
+    <li><em>Home Assistant Container</em>: Standalone container based installation of Home Assistant Core.</li>
```

Comment:
Should this mention the term "Docker"?

Reply:
I don't think so, Docker is only one of the runtimes that can be used, which is explained on the installation page for the container installation.

Reply:
Yeah, I agree on that... However, covers 95% of the use cases. I think it is good, was just suggesting on for example:

```suggestion
    <li><em>Home Assistant Container</em>: Standalone container-based installation of Home Assistant Core (ex: Docker).</li>
```

Or something.